layouts/workshop/single.html

```diff
@@ -0,0 +1,112 @@
+{{ define "main" }}
+
+{{ $path := split $.Source.File.Path .Site.Params.PathSeparator }}
+{{ $event_slug := index $path 1 }}
+{{ $e :=  (index $.Site.Data.events $event_slug) }}
+{{ $city_slug := substr $e.name 5 }}
+
+
+<div class = "row">
+    <div class = "col-md-5 offset-md-1">
+    <h2 class="talk-page">{{ .Title }}</h2>
+
+    {{ if isset .Params "youtube" }}
+      {{ if ne .Params.youtube "" }}
+        <div class="row">
+            <div class="col-md-12 youtube-video">
+              <div class="embed-responsive embed-responsive-16by9">
+              <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/{{ .Params.youtube }}">
+              </iframe>
+              </div>
+            </div>
+          </div>
+        {{ end }}
+    {{ end }}
+    <br />
+    <h3 class="talk-page">Workshop Abstract</h3>
```

Comment:
I also wonder if the only real difference between a talk and a workshop is this - if so, we could probably DRY a lot of this up into partials, since these two templates are mostly copy-pasta from each other.